web/styles/settings.css

```diff
@@ -1995,3 +1994,20 @@ $option_title_width: 180px;
             0 0 8px hsl(206deg 80% 62% / 60%);
     }
 }
+
+#new_stream_can_remove_subscribers_group_id_widget_container,
+#can_remove_subscribers_group_id_widget_container {
+    margin-bottom: 10px;
```

Comment:
An explanation of all the values in these changes to `settings.css` would be good to preserve too :)

Reply:
Done.